### ISSUE TITLE
generate .param file

### ISSUE THREAD

**Issue by tiralonghipol** [PX4/pyulog#104]
ulog_params only shows the list of parameters extracted from the ulog
how to generate back the .params file accordingly? is it something doable?

**Comment by bkueng** [comment]
Yes, you can specify the output format with `--format qgc`: https://github.com/PX4/pyulog/blob/main/pyulog/params.py#L34
